src/routes/blog/index.svelte

```diff
@@ -193,6 +193,12 @@
           }
         }
 
+        a > h2 {
+          overflow: hidden;
+          text-overflow: ellipsis;
+          min-width: 0;
```

Comment:
why min width here? default value for min-width is already 0 and doesnt seem to affect anything when i comment out